packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -55,6 +55,14 @@ export type ExecutionEngineHttpOpts = {
    * +-5 seconds interval.
    */
   jwtSecretHex?: string;
+  /**
+   * A identifier string passed by user in cli arg that will be included in `id` field in jwt claims
```

Comment:
```suggestion
   * An identifier string passed as CLI arg that will be set in `id` field of jwt claims
```

CLI all uppercase seems most consistent with existing code, ideally, in comments should probably also write JWT instead of jwt (does not seem to be case in existing comments, but could consider updating those in this PR)